kalite/distributed/static/js/distributed/distributed-server.js

```diff
@@ -77,6 +80,72 @@ var StatusModel = Backbone.Model.extend({
         return (new Date(new Date() - this.get("client_server_time_diff"))).toISOString().slice(0, -1);
     },
 
+    login: function(username, password, facility, callback) {
+        var self = this;
+
+        data = {
+            username: username || "",
+            password: password || "",
+            facility: facility || ""
+        };
+
+        $.ajax({
+            url: USER_URL + "login/",
+            contentType: 'application/json',
+            dataType: 'json',
+            type: 'POST',
+            data: JSON.stringify(data),
+            success: function(data, status, response) {
+                if (data.redirect) {
+                    window.location = data.redirect;
+                } else {
+                    // TODO (rtibbles) Reinstate the code below once
+                    // the front end app responds better to statusModel changes
```

Comment:
It will require some refactoring of the front end code (particularly exercise, video, and content views) in order to allow them to better handle user data when the statusModel changes.